lib/cisco_node_utils/interface.rb

```diff
@@ -322,6 +322,18 @@ def default_ipv4_arp_timeout
       config_get_default('interface', ipv4_arp_timeout_lookup_string)
     end
 
+    def ipv4_forwarding
+      config_get('interface', 'ipv4_forwarding', @name)
```

Comment:
The merge of feature/grpc changed all of the interface YAML to use key-value instead of printf - so this will need to be `... name: @name`, and below `... name: @name, state: state ? '' : 'no'`